.pre-commit-config.yaml

```diff
@@ -4,7 +4,7 @@ repos:
     hooks:
     - id: black
   - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: v0.0.241
+    rev: v0.0.267
```

Comment:
Since the update introduces so big changes and makes the diff unreadable, let's keep the old version and we can update in a later time if needed.